src/Nest/Search/Suggesters/CompletionSuggester/CompletionField.cs

```diff
@@ -4,16 +4,29 @@
 namespace Nest
 {
 	/// <summary>
-	/// Convenience class for use when indexing completion fields.
+	/// A completion field
 	/// </summary>
+	/// <remarks>
+	/// Convenience class for use when indexing completion fields.
+	/// </remarks>
 	public class CompletionField
 	{
+		/// <summary>
```

Comment:
Hate the hate on adding xmldocs but not fan of these style of SandCastle'esque summaries that feel generated. I rather have none instead. 

Further more to continue my nitpick for small summaries can we have a single line

```chsarp
//// <summary> The summary </summary>
```

Reply:
> for small summaries can we have a single line

I can do this. We'll also need to tweak our formatting configuration which always seems to format on multiple lines